packages/core/src/core/rsbuild.ts

```diff
@@ -221,6 +222,27 @@ export const prepareRsbuild = async (
                 ...(config.module.parser.javascript || {}),
               };
 
+              if (coverage?.enabled && coverage?.provider === 'istanbul') {
+                const swcPluginCoverageInstrument = `${process.cwd()}/node_modules/@rstest/coverage-istanbul/node_modules/swc-plugin-coverage-instrument/target/wasm32-wasip1/release/swc_plugin_coverage.wasm`;
```

Comment:
This path should be imported from `@rstest/coverage-istanbul` rather than hard-coded in rstset.

And `target/wasm32-wasip1/release/swc_plugin_coverage.wasm` is the default export of `swc-plugin-coverage-instrument`

<img width="635" height="132" alt="image" src="https://github.com/user-attachments/assets/e25fe031-2dc1-4a14-af32-3f188efaeccc" />